var/spack/repos/builtin/packages/py-bigdft/package.py

```diff
@@ -26,8 +29,11 @@ class PyBigdft(PythonPackage):
     depends_on("python@3.0:", type=("build", "run"))
```

Comment:
Versions 1.9.4-1.9.5 require a newer version of `python`:

```
requires-python = ">=3.6"
```

There appear to be missing dependencies as well per their `setup.py` files.  For 1.9.3:

```
      install_requires=['ase', 'matplotlib', 'numpy'],
```

and versions 1.9.4-1.9.5:

```
      install_requires=['numpy', 'scipy', 'PyFutile'],'
```